### ISSUE TITLE
JS error on mousemove, $.throttle

### ISSUE THREAD

**Issue by DavidThomas48** [phpmyadmin/phpmyadmin#16052]
### Describe the bug

js error message on screen

### To Reproduce
can't

### Screenshots

![image](https://user-images.githubusercontent.com/1977575/77822859-a223df80-714a-11ea-8e6e-cfc36b4a6b15.png)

![image](https://user-images.githubusercontent.com/1977575/77822884-cc759d00-714a-11ea-933b-033e268185a1.png)

### Server configuration
Win10 64 XAMPP
### Client configuration
Chrome

**Comment by Tithugues** [comment]
@DavidThomas48 , would it be please possible to get the screenshot of the whole page with the errors displayed?
Can you please confirm that you have the error on the home page?

**Comment by DavidThomas48** [comment]
Hi Hugues,

If it happens again I will do that. I can tell you that it happens when I resize the Routines editor dialog.

David

**Comment by DavidThomas48** [comment]
Hello Hugues ,

I just experienced the problem and it happened when I dragged the divider to the right to widen the left sidebar (database navigation tree). I had previously dragged it left to make the sidebar very narrow, then done some other things.

Hope that helps.

David

**Comment by DavidThomas48** [comment]
Hi again,
Re "Can you please confirm that you have the error on the home page?" - no this is not the case.  It happens when I am working in tables or routines. I am rarely on the home page.

**Comment by williamdes** [comment]
Reading the code it seems like `js/vendor/jquery/jquery.debounce-1.0.5.js` did throttle the call and then invoke it but forgetting the event data

**Comment by williamdes** [comment]
@DavidThomas48 does this patch solve this issue for you ?
```diff
diff --git a/js/navigation.js b/js/navigation.js
index 169d4e40592..43f104ce2f9 100644
--- a/js/navigation.js
+++ b/js/navigation.js
@@ -1298,10 +1298,12 @@ Navigation.ResizeHandler = function () {
      */
     this.mousemove = function (event) {
         event.preventDefault();
-        var pos = event.data.resize_handler.getPos(event);
-        event.data.resize_handler.setWidth(pos);
-        if ($('.sticky_columns').length !== 0) {
-            Sql.handleAllStickyColumns();
+        if (event.data.resize_handler) {
+            var pos = event.data.resize_handler.getPos(event);
+            event.data.resize_handler.setWidth(pos);
+            if ($('.sticky_columns').length !== 0) {
+                Sql.handleAllStickyColumns();
+            }
         }
     };
     /**
```

**Comment by DavidThomas48** [comment]
Hello William,

 

I am working on a WordPress site that contains only one copy of navigation.js which is only 3.66Kb and does not contain Navigation.ResizeHandler = function () {…

 

I should add that the error happens very rarely – I had not seen it for well over a month until the other day and I am working in pma every day.

 

Regards

 

David Thomas

**Comment by DavidThomas48** [comment]
Sorry William – senior moment – just making the patch now but I really won’t know if it has fixed the problem because of the rarity of the error.

DT

 

From: DAVID THOMAS [mailto:david.thomas@elliott-thomas.com.au] 
Sent: Monday, 8 June 2020 11:09 AM
To: 'phpmyadmin/phpmyadmin'; 'phpmyadmin/phpmyadmin'
Cc: 'Mention'
Subject: RE: [phpmyadmin/phpmyadmin] JS error on mousemove (#16052)

 

Hello William,

 

I am working on a WordPress site that contains only one copy of navigation.js which is only 3.66Kb and does not contain Navigation.ResizeHandler = function () {…

 

I should add that the error happens very rarely – I had not seen it for well over a month until the other day and I am working in pma every day.

 

Regards

 

David Thomas

**Comment by williamdes** [comment]
Hi

From what you commented I can understand you where probably no up to date :)
I am not sure that I will post my patch, can you let me know if you where using a 5.0 version?

**Comment by DavidThomas48** [comment]
I have installed the patch on my server. If the error hasn’t happened by August, it’s probably fixed!

D

**Comment by DavidThomas48** [comment]
I just realised, I have been working online with a host site recently, and they are running pma 4.9.1. I am fairly sure that the error occurred on my machine (pma 5.0.2), but it is possible that the error occurred on that site. Sorry to be vague about this. Doesn’t the error  report give you the pma  version?

David

 

From: DAVID THOMAS [mailto:david.thomas@elliott-thomas.com.au] 
Sent: Monday, 8 June 2020 6:25 PM
To: 'phpmyadmin/phpmyadmin'
Subject: RE: [phpmyadmin/phpmyadmin] JS error on mousemove, $.throttle (#16052)

 

I have installed the patch on my server. If the error hasn’t happened by August, it’s probably fixed!

D
